benchmarks/dynamo/common.py

```diff
@@ -284,6 +285,36 @@ class CI(NamedTuple):
 }
 
 
+@functools.lru_cache(maxsize=1)
```

Comment:
Should we increase this cache max size? i.e. is there any case where we call this function twice with different filenames?

Reply:
"call this function twice with different filenames" - that's unlikely and in any way is not really a problem - the old value will be evicted from the cache.

Reply:
Maybe we can just keep `maxsize=None` here and be done with it?

Reply:
I think `maxsize=1` is fine and reflects the intended usage.